docs/quickstart.mdx

```diff
@@ -35,14 +35,21 @@ Our test will succeed if the model outputs valid JSON.
   </Step>
 
   <Step title="Run the test">
+    This step requires an `OPENAI_API_KEY` to authenticate with OpenAI.
```

Comment:
I was thinking, given this is a quick-start we should suggest setting variables in the shell command itself. 

and post one successful run we can suggest to setup `.env ` file.

wdyt ?

Reply:
Yeah sounds good! Just wanted to give context on the env variables to the user. 

Where do you think is a best place to suggest about setting up a `.env` file? or do we even need to suggest this?

Reply:
I think at the end of this section mention a link for setting an environment variable in .env to make the CLI command cleaner (`npx @empiricalrun/cli run`)